notebookgen.js

```diff
@@ -30,7 +31,9 @@ function walk (_path, depth) {
       ans += '\n' + section[depth] + file + '}\n' + walk(f, depth + 1)
     } else if (path.extname(f) in extensions) {
       ans += '\n' + section[depth] + file.split('.')[0] + '}\n'
-      if (path.extname(f) !== '.tex') {
+      if (path.extname(f) === '.png') {
```

Comment:
It might be interesting to support other file formats too.

We might have a set of "image file extensions" and then check if the current one is in the set.

WDYT?